src/domains/transaction/components/UsernameRegistrationDetail/UsernameRegistrationDetail.tsx

```diff
@@ -27,8 +27,6 @@ export const UsernameRegistrationDetail = ({ isOpen, transaction, onClose }: Tra
 		>
 			<TransactionSender address={transaction.sender()} network={transaction.wallet().network()} border={false} />
 
-			{/* @TODO: Use new icons for username registration & resignation types when available.
```

Comment:
this would need a custom icon instead of the delegate icon it relies on right now